.github/workflows/test-action.yml

```diff
@@ -43,6 +43,9 @@ jobs:
 
       - name: Execute coverage
         run: elixir --sname aeternity@localhost -S mix coveralls
+        env:
+          PORT: 4002
+          WS_PORT: 4003
```

Comment:
This is not necessary because tests don't use the http server and ws is only accessed by test cases.

Reply:
If I don't add them, these tests fail: https://github.com/aeternity/ae_mdw/blob/master/test/ae_mdw/websocket/socket_handler_test.exs#L10